experiment/autobumper/bumper/bumper.go

```diff
@@ -288,11 +288,22 @@ func Run(o *Options) error {
 		}
 
 		// Check to see if the proper fork exists and if it does not, create one.
-		// TODO(mpherman): Handle case where account has repo of same name as one we are trying to autobump but is not a fork of it (e.g. test-infra)
-		_, err := gc.EnsureFork(o.GitHubLogin, o.GitHubOrg, o.GitHubRepo)
+		forkName, err := gc.EnsureFork(o.GitHubLogin, o.GitHubOrg, o.GitHubRepo)
 		if err != nil {
 			return fmt.Errorf("fork needed for autobump does not exist. unable to create new fork. %v", err)
 		}
+		// If a new fork was created with a name other than o.RemoteName
+		if forkName != o.RemoteName {
+			var updateRequest = github.RepoUpdateRequest{
+				RepoRequest: github.RepoRequest{
+					Name: &o.RemoteName,
+				},
+			}
+			_, err := gc.UpdateRepo(o.GitHubLogin, forkName, updateRequest)
```

Comment:
After moving repo from `orgA/repoA` to `orgA/repoB`, Github redirects all requests against `orgA/repoA` to `orgA/repoB`, does this affect this use case? The reason for this question it that I  imagine the reason for this improvement is that the `forkName` is supposed to be repurposed to be used by something else, would later on querying `forkName` from somewhere else get you unexpected results?

Reply:
forkName is returned by EnsureFork. EnsureFork checks to see if a fork already exists for a given repo, if not creates one, and then returns the name of the fork. So if we had a istio-test-infra repo that was a fork of istio/test-infra Ensure fork would return "istio-test-infra." If we did not already have a istio fork, but had another test-infra fork, Ensurefork would create a fork and return "test-infra-1". forkName is only used to compare this with what the config file says the remote name is and if they don't match it will change the name of the fork with UpdateRepo. forkName is not used ever again. Most of the time forkName == remoteName. The only case where this won't be true is on the first run of the autobumper if the fork does not already exist.

Reply:
I added a log info when the fork name gets changed, and elaborated on the Options.RemoteName comment to hopefully make this more clear as well.

Reply:
ah. I see, totally makes sense now. Thank you!